poseidon2/src/external.rs

```diff
@@ -191,10 +232,16 @@ impl<T, const WIDTH: usize> ExternalLayerConstants<T, WIDTH> {
         Self::new(initial_consts, terminal_consts)
     }
 
+    /// Get a reference to the list of initial round constants.
+    ///
+    /// These are used in the first half of the external rounds.
     pub const fn get_initial_constants(&self) -> &Vec<[T; WIDTH]> {
         &self.initial
     }
 
+    /// Get a reference to the list of terminal round constants.
+    ///
+    /// These are used in the second half (final rounds) of the external layer.
```

Comment:
Just to keep narrative consistency.
```suggestion
    /// These are used in the second half (terminal rounds) of the external layer.
```